src/bin/travis/resources/funnel.conf

```diff
@@ -1,10 +1,10 @@
 HttpPort: 9000
 Storage:
-  - Local:
-      AllowedDirs:
-        - /home/
-        - /cromwell-executions
-        - /tmp/
+  Local:
+    AllowedDirs:
+      - /home/
+      - /cromwell-executions
+      - /tmp/
```

Comment:
hrm?

Reply:
I don't know how it was working before but this conf was outdated, they changed the format and the `Storage` thing is an object not a list